includes/cli/anonymize.php

```diff
@@ -19,10 +21,27 @@ class SafetyNet_CLI extends WP_CLI_Command {
 	*
 	*/
 	public function anonymize( $args ) {
+
+		$info = WP_CLI::colorize( '%pThis process will anonymize your current users, orders and customers with dummy data.%n ' );
+		WP_CLI::log( $info );
+
+		WP_CLI::warning( 'Please proceed with caution if you have a site with a large number of users/orders/customers' );
+
+		WP_CLI::confirm( 'Are you sure you want to do this?' );
+
+		WP_CLI::log( '- Copying users to temporary tables ...' );
+		copy_and_clear_user_tables();
```

Comment:
Do any of these functions return data that could be shown to the user? e.g error handling, success message, count of changes rows etc?